docs/src/devdocs/reference_cells.md

```diff
@@ -4,6 +4,29 @@ The reference cells are used to i) define grid cells, ii) define shape functions
 define quadrature rules. The numbering of vertices, edges, faces are visualized below. See also
 [`FerriteViz.elementinfo`](https://ferrite-fem.github.io/FerriteViz.jl/dev/api/#FerriteViz.elementinfo).
 
+## Numbering and directions of entities
+
+The local numbering of vertices, edges, and faces, on the reference cells, specifies, for
+example, the cell node order, and the order in which interpolations distribute their DoFs.
+It is important the the same convention is used everywhere to not cause mismatches. The
+convention adopted by Ferrite.jl is documented below.
+
+##### Vertices
+Vertices (nodes) are numbered based on their coordinate in the reference system in *reverse*
+lexicographic ordering, i.e. the inner-most numbering loop is for the ``ξ₁`` direction. See
+e.g. [`RefQuadrilateral`](@ref Ferrite.RefQuadrilateral) for an example.
+
+##### Edges
+Edges are defined by the 2-tuple of vertices it connects, starting with the smallest vertex
+number. To enumerate the edges of the reference cell lexicographic ordering is used on the
+vertex tuples. See e.g. [`RefQuadrilateral`](@ref Ferrite.RefQuadrilateral) for an example.
```

Comment:
This convention causes a seemingly random order on edges and faces, perhaps it is better to use coordinate ordering for the midpoint of the edge face instead?

Reply:
Maybe lexicographic ordering in the reference coordinates? I think this should be almost what we already do.

Reply:
You can interpret your coordinate vector as a word and the coordinates (floats) as a generator for your alphabet.

Reply:
I misread what you wrote. I think we already do something like this on master for the interpolation, right? https://github.com/Ferrite-FEM/Ferrite.jl/blob/master/src/interpolations.jl#L696-L712

Reply:
That would stay (but we would permute in `Ferrite.value`). This PR is for the ordering of the edges themselves, which dof order will match

Reply:
Yes, but the facedofs/edgedofs in the interpolations are lexicographically ordered w.r.t. face/edge center coordinates, right?

Reply:
Those functions just connect dof index with entity index, so wouldn't change: We would still want e.g. dof 1 in vertex 1 and first edge dof in edge 1. This PR is for potentially redefining which vertex is vertex 1, and which face is face 1, etc.